src/pages/IDOPage/IDOPage.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import useStyle from './styles'
+import { Grid, Typography, Box, CardMedia, Button, Input } from '@material-ui/core'
+import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
+import icons from '@static/icons'
+import { Status } from '@reducers/solanaWallet'
+import { useSelector } from 'react-redux'
+import { status } from '@selectors/solanaWallet'
+
+const IDOPage = () => {
```

Comment:
don't hardcode data inside components, use props and set mock data inside storybook story

Reply:
also task was only for component and story with mock data, page and connection to redux were not in scope